src/documents/APIChangesFrom1_5To2_0.md

```diff
@@ -0,0 +1,136 @@
+---
+title: API Changes from 1.5 to 2.0
+layout: default
+---
+
+# API Changes from 1.5 to 2.0 #
+
+## Framework Changes ##
+
+### Preferences Framework ###
+
+#### Namespace Changes ####
+
+Rename "fluid.prefs.enactors" to "fluid.prefs.enactor"
+
+#### Component Grade Changes ####
```

Comment:
Are the component grade changes and the model sharing changes restricted to the Prefs Framework, or do they affect all of Infusion, @jobara?